tssc/step_implementer.py

```diff
@@ -224,8 +224,9 @@ def _validate_runtime_step_config(self, runtime_step_config):
         """
         missing_required_config_keys = []
         for required_config_key in self.required_runtime_step_config_keys():
-            if (required_config_key not in runtime_step_config) or \
-                    (not runtime_step_config[required_config_key]):
+            if ((required_config_key not in runtime_step_config) or \
```

Comment:
[`and` is higher precedence than `or`](https://docs.python.org/3/reference/expressions.html#operator-precedence), which I think is probably intuitive and is probably what you meant here, so take this comment with a grain of salt.  But for future readers/maintainers of this code, I tend to add the extra parens around what I want to be evaluated first to try and be extra clear.

Reply:
This was an oversight. Corrected. Thanks for catching it @jkeam .